lib/index.ts

```diff
@@ -12,6 +12,10 @@ export import StreamInfo = require("./StreamInfo");
 export import TrackInfo = require("./TrackInfo");
 export import RTCPFeedbackInfo = require("./RTCPFeedbackInfo");
 export import TrackEncodingInfo = require("./TrackEncodingInfo");
+export import RIDInfo = require("./RIDInfo");
+export import SimulcastInfo = require("./SimulcastInfo");
+export import SimulcastStreamInfo = require("./SimulcastStreamInfo");
+export import DataChannelInfo = require("./DataChannelInfo");
```

Comment:
this PR also exports these classes that were not exported before. I'm not sure why that was the case :sweat_smile: but if you want to keep them hidden, i can revert this change